src/js/view/controls/nextuptooltip.js

```diff
@@ -124,15 +125,15 @@ export default class NextUpTooltip {
             // Set title
             this.title = this.content.querySelector('.jw-nextup-title');
             const title = nextUpItem.title;
+            // createElement is used to leverage 'textContent', to protect against developers passing a title with html styling.
             this.title.innerText = title ? createElement(title).textContent : '';
 
             // Set duration
-            if (nextUpItem.duration) {
+            const duration = nextUpItem.duration;
+            if (duration) {
                 this.duration = this.content.querySelector('.jw-nextup-duration');
-                const duration = nextUpItem.duration;
-                this.duration.innerText = duration ? createElement(duration).textContent : '';
+                this.duration.innerText = typeof duration === 'number' ? timeFormat(duration) : duration;
```

Comment:
What if `duration` is `undefined`? Have you tested this?

Reply:
the line would never run because we're in an `if (duration)` block